packages/client/utils/makeTemplateDescription.ts

```diff
@@ -1,26 +1,46 @@
 import graphql from 'babel-plugin-relay/macro'
+import {useFragment} from 'react-relay'
 import {readInlineData} from 'relay-runtime'
 import {makeTemplateDescription_template$key} from '../__generated__/makeTemplateDescription_template.graphql'
+import {makeTemplateDescription_viewer$key} from '../__generated__/makeTemplateDescription_viewer.graphql'
+import {TierEnum} from '../__generated__/SendClientSegmentEventMutation.graphql'
 import relativeDate from './date/relativeDate'
 
 const makeTemplateDescription = (
   lowestScope: string,
-  templateRef: makeTemplateDescription_template$key
+  templateRef: makeTemplateDescription_template$key,
+  viewerRef?: makeTemplateDescription_viewer$key | null,
```

Comment:
In Poker, there are no premium templates. We could pass the `viewerRef` to this function from the poker files so that the Poker public template subtitles say "Free template" instead of "Created by Parabol", but that would involve passing the `viewerRef` through many components, as you can see in this PR on the retro side. 

Instead, I'd rather keep the Poker public template subtitles as they are for now. When we remove the feature flag (https://github.com/ParabolInc/parabol/issues/7413), the Poker subtitles will change to "Free template", just like the retro templates.

Reply:
Hmm we'll actually need access to the `featureFlags` and therefore the `User` object in Poker to show the custom template team & org copy. I'll add it to this PR - it'll probably bump it up to a large, but as other PRs will build off of this one, I think it's the right move. Removing my request for a review until this is done